### ISSUE TITLE
TUTORIAL 09 MyToken.sol

### ISSUE THREAD

**Issue by jjmr007** [willitscale/learning-solidity#32]
function "approve" shuoldn't be something like this?

    `function approve(address _spender, uint _value) public returns (bool success) {
       
        if (_value < 0 && __balanceOf[msg.sender] >= _value) {
        __allowances[msg.sender][_spender] = _value;
        return true;
        }
        return false;
    }`
line 49, file MyToken.sol
thanks! =)

**Comment by YannyD** [comment]
I don't think so.  He put these constraints in the 'transferFrom' function.  You'll see those, along with some others, on lines 36-39.
